package.json

```diff
@@ -8,10 +8,10 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "browserslist": [
-    "chrome >= 61",
-    "firefox >= 60",
-    "safari >= 11",
-    "edge >= 16"
+    "chrome >= 80",
+    "firefox >= 76",
+    "safari >= 13",
+    "edge >= 80"
```

Comment:
this is also at 98 now
and 79 was when they switched to chromium, so we aren't staying at a low version to account for that